src/System Application/App/Email/src/Message/EmailAttachments.Page.al

```diff
@@ -53,15 +53,38 @@ page 8889 "Email Attachments"
     {
         area(Processing)
         {
+            fileuploadaction(UploadMultiple)
+            {
+                ApplicationArea = All;
+                Image = Attach;
+                Caption = 'Add files';
+                ToolTip = 'Attach files, such as documents or images, to the email.';
+                Scope = Page;
+                Visible = IsEmailEditable;
 
+                trigger OnAction(files: List of [FileUpload])
```

Comment:
Not first captial letter of parameter on trigger "files" just points to my eye...